src/Erd/Config.hs

```diff
@@ -167,6 +169,14 @@ opts =
                 "EDGE")
       (printf "Select one type of edge:\n%s"
               (intercalate ", " $ M.keys edges))
+  , O.Option [] ["no-html"]
```

Comment:
Finding a more specific name for the argument.

Reply:
I agree on using your proposed dot-entity ( #72 )

Reply:
I guess we should change the name inside Config data type from _nothtml_ to _dotentity_